styles.css

```diff
@@ -62,9 +62,291 @@ body {
 .social-media-icons {
   height: 30px;
   display: flex;
+  margin: 8px;
 }
 
 .social-media-icons a {
   text-decoration: none;
-  margin: 8px;
+  align-items: center;
+  margin: 6px;
+}
+
+.social-media-icons img {
+  padding: 6px;
+}
+
+.works {
+  display: grid;
+  flex-direction: column;
+  align-items: center;
+  height: 2550px;
+  padding: 114px 24px;
+}
+
+.card-works {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 16px;
+  width: 327px;
+  height: 630px;
+  left: calc(50% - 327px / 2);
+  top: 100px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  box-sizing: border-box;
+  border-radius: 16px;
+  margin: 30px 0;
+}
+
+.snapshoot-portfolio {
+  height: 220px;
+  margin: 12px 0;
+}
+
+.portfolio-snapshoot {
+  border-radius: 8px;
+}
+
+.left-block {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 0;
+  width: 295px;
+  height: 288px;
+  left: 16px;
+  top: 248px;
+  margin: 12px 0;
+}
+
+.project-title {
+  font-weight: 700;
+  font-size: 24px;
+  color: #344563;
+}
+
+.summary-frame {
+  display: flex;
+  flex-direction: row;
+  padding: 0;
+}
+
+.client {
+  margin-right: 10px;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+}
+
+.role {
+  margin-right: 10px;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.year {
+  margin-right: 10px;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.project-description p {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.tags {
+  display: flex;
+  flex-direction: row;
+}
+
+.project-button {
+  position: relative;
+  display: inline-block;
```

Comment:
[OPTIONAL] you can use `flexbox + margin/padding` to layout your element instead of using `position right/top`, this will make your project more professional.